common/src/stack/command/stack/commands/report/host/__init__.py

```diff
@@ -84,7 +84,9 @@ def run(self, param, args):
 				for option in options:
 					if option.strip() == 'shortname':
 						h['shortname']= True
-			hosts[host].append(h)
+						
+                        if self.validateHostInterface(host, h, aliases):
```

Comment:
Don't mix tabs and spaces for indentation!

In stacki we only use tabs.  You can uses spaces if necessary to align text (for example a continued line).